modules/deploy/pages/deployment-option/cloud/managed-connectors/create-snowflake-connector.adoc

```diff
@@ -183,6 +185,9 @@ Additional errors and corrective actions follow.
 
 |*Config:value.converter has provided value:com.snowflake.kafka.connector.records.SnowflakeJsonConverter. If ingestionMethod is:snowpipe_streaming, Snowflake Custom Converters are not allowed.*
 |Use `STRING` for the `Kafka message value format`.
+
+|* Ignoring invalid task provided offset mytopic-12/OffsetAndMetadata{offset=7182, leaderEpoch=null, metadata=''} -- not yet consumed, taskOffset=7182 currentOffset=6804"}*
+|Snowflake tables store metadata containing offset info. Likely this topic has been dropped and recreated since the initial sync to Snowflake causing issue with mismatch of offsets. When deleting/recreating Redpanda topics that have already been sent data to Snowflake,you need to delete the corresponding Snowflake table as well.
```

Comment:
```suggestion
|Snowflake tables store metadata containing offset information. It is likely that this topic has been dropped and recreated since the initial sync to Snowflake, and this may cause a mismatch of offsets. When deleting and/or recreating Redpanda topics that have already sent data to Snowflake, you must delete the corresponding Snowflake table as well.
```